### PR TITLE
aptconf syntax update

### PR DESCRIPTION
**Pull request by 532910**
Signed-off-by: Sergio E. Nemirowski <sergio@outerface.net>

### CODE REVIEW

**Comment by 532910** [comment]
Yann Amar was notified by e-mail.

**Comment by 532910** [comment]
keywords are updated from https://github.com/mvo5/unattended-upgrades

File: runtime/syntax/aptconf.vim [hunk 1]
```diff
@@ -396,10 +396,13 @@ syn cluster	aptconfSynaptic_ contains=aptconfSynaptic,
...
-	\ Automatic-Reboot-WithUsers InstallOnShutdown Mail MailOnlyOnError
-	\ MinimalSteps Origins-Pattern Package-Blacklist
-	\ Remove-Unused-Dependencies
+	\ Allow-APT-Mark-Fallback Allow-downgrade AutoFixInterruptedDpkg
+	\ Automatic-Reboot Automatic-Reboot-Time Automatic-Reboot-WithUsers
+	\ Debug InstallOnShutdown Mail MailOnlyOnError MailReport MinimalSteps
+	\ OnlyOnACPower Origins-Pattern Package-Blacklist
+	\ Remove-New-Unused-Dependencies Remove-Unused-Dependencies
+	\ Remove-Unused-Kernel-Packages Skip-Updates-On-Metered-Connections
+	\ SyslogEnable SyslogFacility
```
Comment (quidame) [inline runtime/syntax/aptconf.vim]: the `Verbose` keyword is already defined elsewhere, but I prefer to be exhaustive in each group of keywords, so if someone wants to change something in keywords handling, each group can be used as its own reference.
```suggestion
	\ SyslogEnable SyslogFacility Verbose
```

**Review by quidame — CHANGES_REQUESTED** [review]
@532910 thanks for updating this !

**Comment by quidame** [comment]
@532910 could you also update the `Last Change` field in the file's header, please ?

**Comment by 532910** [comment]
Done.

**Review by quidame — APPROVED** [review]

**Comment by brammool** [comment]
I'll include it, thanks.

**Pull request closed without merging**